google/resource_bigtable_instance.go

```diff
@@ -0,0 +1,193 @@
+package google
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+
+	"cloud.google.com/go/bigtable"
+	"golang.org/x/net/context"
+)
+
+func resourceBigtableInstance() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceBigtableInstanceCreate,
+		Read:   resourceBigtableInstanceRead,
+		Delete: resourceBigtableInstanceDestroy,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"zone": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"display_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+
+			"num_nodes": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				ForceNew:     true,
```

Comment:
We can't update it with the client as-is, or even read it for that matter. When the client supports updating this value, we can remove `ForceNew`.